.circleci/base_config.yml

```diff
@@ -16,15 +16,18 @@ executors:
       image: ubuntu-2204:2022.10.1
 
 parameters:
+  develop_branch_deploy:
+    type: boolean
+    default: false
   run_dev_deployment:
     type: boolean
     default: false
```

Comment:
This seems clean and fine, but could you have used `run_dev_deployment` and hooked into the existing `staging-deployment` workflow?

Reply:
I did try that at first, but it seemed to tie in things I did not want as well. I tried adding some logic, but it got pretty messy and still didn't do what I wanted... so I gave up and created a different flag to operate off of.